Information/Design.md

```diff
@@ -62,8 +62,25 @@
 
 **070 The bot shall be able to provide a list of products the vendor sells**
 
-
-
+**Requirement 080: Upon user input, the bot shall be able to display the currently active events**  
+  Command: `-events`  
+  Return: A list of active events.  
+  
+**Requirement 090: The bot shall be able to display what monsters appear and at what difficulty within the ecosystem specified by the user**  
+  Command: `-location [ecosystem][rank]`  
+  Return: The monster availabale in the ecosystem at the rank specified by the user.  
+  
+**Requirement 100: The bot shall be able to provide information about permanently equipped tools**  
+  Command: `-hunter tools`  
+	Return: Small description of each fixed hunter tool (i.e. scout flies, clutch claw, etc)  
+
```

Comment:
For commands that take parameters, or arguments, whatever. Will the command still work if the user inputs five arguments instead of the desired arguments in the case of **REQ 120**??? If it does still work will it only except the first two arguments or the last two arguments. Consider the case when two arguments are required for the command name to function properly. What if the user inputs only one of these two required arguments. Does it then return an error message, or nothing at all, or still the desired monsters that spawn in the area at that level?  What if you get none of these expected arguments to the command name. What then???    Then you need to document why you chose this behavior over other potential options, for these inconsistent user inputs, or a lack of knowledge of how the program works. Thus the user may input incorrect values.   Which brings me to an important question. For this commands that are being designed. Do you require the user to now anything about the game, or how the commands work at all. Or should you print a helpful message to educated a novice user of the Monster Hunter World Bot when they use the commands for the first time and input wrong values accordingly.